src/grokcore/view/templatereg.py

```diff
@@ -162,6 +163,9 @@ def _register_template_file(self, module_info, template_path):
                               "for pt files. Now the file '%s' in '%s' "
                               "cannot be rendered" %
                               (template_file, template_dir), UserWarning, 2)
+            elif extension == '':
```

Comment:
Thanks @thefunny42 for picking this one up. 

Purely for this PR this line of code can go, correct?